### PR TITLE
SUBMARINE-343. Update HowToRun.md due to new docker images

### PR DESCRIPTION
**Pull request by kevin85421**
### What is this PR for?
Update /docs/workbench/HowToRun.md due to new docker images

### What type of PR is it?
Documentation

### What is the Jira issue?
https://issues.apache.org/jira/projects/SUBMARINE/issues/SUBMARINE-343?filter=allissues

### How should this be tested?
https://travis-ci.org/apache/submarine?utm_medium=notification&utm_source=github_status

### Screenshots (if appropriate)
* version Angular
截圖 2020-01-19 下午1 30 40
* version Vue
截圖 2020-01-19 下午1 33 15

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: docs/workbench/HowToRun.md [hunk 1]
```diff
@@ -13,67 +13,145 @@
...
+1. Angular (default)
+2. Vue (This is the old version, and it will be replaced by version Angular in the future.)
+#### (WARNING: Please restart a new **incognito window** when you switch to different versions of Submarine Workbench) 
+### Launch the Submarine Workbench(Angular)
+* It should be noted that since Submarine Workbench depends on the Submarine database, so you need to run the docker container of the Submarine database first.
+```
+docker run -it -p 3306:3306 -d --name submarine-database -e MYSQL_ROOT_PASSWORD=password apache/submarine:database-0.3.0-SNAPSHOT
+docker run -it -p 8080:8080 -d --link=submarine-database:submarine-database --name submarine-server apache/submarine:server-0.3.0-SNAPSHOT
+```
+* The login page of Submarine Workbench will be shown in ```127.0.0.1:8080```.
```
Comment (liuxunorg) [inline docs/workbench/HowToRun.md]: Please change `127.0.0.1:8080` to `http://127.0.0.1:8080`.

File: docs/workbench/HowToRun.md [hunk 2]
```diff
@@ -13,67 +13,145 @@
...
+```
+./bin/submarine-daemon.sh restart
+```
+*  Step7: Launch the submarine-server container
+```
+docker start submarine-server
 ```
-docker run -it -p 3306:3306 -d --name submarine-data -e MYSQL_ROOT_PASSWORD=password apache/submarine:database-0.3.0
-docker run -it -d --link=submarine-data:submarine-data --name submarine-server apache/submarine:submarine-0.3.0
+*  Step8: Open a new **incognito window(not a tab)** and check ```127.0.0.1:8080```
```
Comment (liuxunorg) [inline docs/workbench/HowToRun.md]: Please change `127.0.0.1:8080` to `http://127.0.0.1:8080`.

File: docs/workbench/HowToRun.md [hunk 3]
```diff
@@ -13,67 +13,145 @@
...
+*  Step2: Modify the value of the configutation **workbench.web.war** in conf/submarine-site.xml from "../workbench-web.war" to "../workbench-web-ng.war".
+*  Step3: Restart the Submarine Server
+```
+./bin/submarine-daemon.sh restart
+```
+*  Step4: Launch the submarine-server container
+```
+docker start submarine-server
+```
+*  Step5: Open a **new incognito window(not a tab)** and check ```127.0.0.1:8080```
```
Comment (liuxunorg) [inline docs/workbench/HowToRun.md]: Please change `127.0.0.1:8080` to `http://127.0.0.1:8080`.

File: docs/workbench/HowToRun.md (hunk 1 above)
Comment (kevin85421) [reply docs/workbench/HowToRun.md]: Thanks for your review. I have fixed it.

File: docs/workbench/HowToRun.md (hunk 2 above)
Comment (kevin85421) [reply docs/workbench/HowToRun.md]: Thanks for your review. I have fixed it.

File: docs/workbench/HowToRun.md (hunk 3 above)
Comment (kevin85421) [reply docs/workbench/HowToRun.md]: Thanks for your review. I have fixed it.

**Comment by liuxunorg** [comment]
Will merge if no more comments.

**Pull request closed without merging**